docs/guides/validators/security_best_practices.md

```diff
@@ -330,27 +330,58 @@ Once the installation is complete, you can save your configuration using the com
 sudo invoke-rc.d iptables-persistent save
 ```
 
-### Rule for ssh connection
+### One command firewall configuration
```

Comment:
that's definitely not one command but whole four :)

Reply:
we should make it clear that this configuration is for validator, not RPC.

Reply:
fixed